data/preferences.ui

```diff
@@ -49,6 +49,8 @@
                         <property name="use_underline">True</property>
                         <property name="xalign">0.5</property>
                         <property name="draw_indicator">True</property>
+                        <property name="has_tooltip">True</property>
+                        <property name="tooltip_text" translatable="yes">Work only on GNOME</property>
```

Comment:
"Only works on GNOME"

Reply:
Actually I think it would be better if the main description was changed to "Stop tracking on shutdown (GNOME only)".

Reply:
The tooltip appears quite easily here so I think it was enough but I have changed the label.